diff --git a/lua/lazy/help.lua b/lua/lazy/help.lua
index 46b47fd..f2fe6b3 100644
--- a/lua/lazy/help.lua
+++ b/lua/lazy/help.lua
@@ -39,9 +39,9 @@ function M.index(plugin)
 end

 function M.update()
-  if Config.plugins["lazy.nvim"] then
-    vim.cmd.helptags(Config.plugins["lazy.nvim"].dir .. "/doc")
-  end
+  -- if Config.plugins["lazy.nvim"] then
+  --   vim.cmd.helptags(Config.plugins["lazy.nvim"].dir .. "/doc")
+  -- end
   if Config.options.readme.enabled == false then
     return
   end
diff --git a/lua/lazy/manage/task/plugin.lua b/lua/lazy/manage/task/plugin.lua
index cec9762..466a8b8 100644
--- a/lua/lazy/manage/task/plugin.lua
+++ b/lua/lazy/manage/task/plugin.lua
@@ -91,9 +91,9 @@ M.docs = {
   end,
   run = function(self)
     local docs = self.plugin.dir .. "/doc"
-    if Util.file_exists(docs) then
-      self:log(vim.api.nvim_cmd({ cmd = "helptags", args = { docs } }, { output = true }))
-    end
+    -- if Util.file_exists(docs) then
+    --   self:log(vim.api.nvim_cmd({ cmd = "helptags", args = { docs } }, { output = true }))
+    -- end
   end,
 }
